_episodes/05-creating-histograms.md

```diff
@@ -71,56 +71,39 @@ tools we will use to draw the histogram. The statement
 
 loads up the `pyplot` library, and gives it a shorter name, `plt`. 
 
-Next, we use the `cv2.imread()` function to load our image. We use the first 
+Next, we use the `skimage.io.imread()` function to load our image. We use the first 
 command line parameter as the filename of the image, as we did in the 
-[OpenCV Images]({{ page.root }}/03-opencv-images) lesson. The second parameter
-to `cv2.imread()` instructs the function to transform the image into 
+[Skimage Images]({{ page.root }}/03-skimage-images) lesson. The second parameter
+to `skimage.io.imread()` instructs the function to transform the image into 
 grayscale as it is loaded in to the program. Note that this does not change
-the original image. There are OpenCV functions to convert a color image to 
+the original image. There are skimage functions to convert a color image to 
 grayscale, but in cases where the program does not need the color image, we can
 save ourselves some typing by loading the image as grayscale from the outset.
 
-The next salient piece of code is where we ask OpenCV to create the histogram:
+Skimage does not provide a special function to compute histograms, but we can use
+the function `np.histogram` instead:
 
 ~~~
 # create the histogram
-histogram = cv2.calcHist(images = [image], 
-    channels = [0], 
-    mask = None, 
-    histSize = [256], 
-    ranges = [0, 256])
+histogram = np.histogram(image, bins = 256, range = (0, 256))
```

Comment:
I'm quite positive this will not work as expected. `skimage` converts data to `float` when using `as_gray=True` and rescales the original dtype range to `0..1`. How about:
```suggestion
histogram, bin_edges = np.histogram(image, bins=256, range=(0,0, 1.0))
bin_edges *= 256
```
note that I checked the docs for `np.histogram` because I remembered that it does [return more than just counts])https://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram.html) 

The text following this code fragment should be adapted accordingly..